webui/src/lib/components/repositoryCreateForm.jsx

```diff
@@ -15,7 +15,7 @@ const DEFAULT_BLOCKSTORE_VALIDITY_REGEX = new RegExp(`^s3://`);
 
 export const RepositoryCreateForm = ({ config, onSubmit, onCancel, error = null, inProgress = false, sm = 6 }) => {
     const fieldNameOffset = 3;
-    const repoValidityRegex = /^[a-z0-9][a-z0-9-]{2,62}$/;
+    const repoValidityRegex = /^[a-z0-9][a-z0-9-]{1,62}$/;
```

Comment:
I think the pattern was ok - based on "Bucket names must be between 3 (min) and 63 (max) characters long" like s3.

Reply:
Thanks @nopcoder I didn't realize original naming req's were based on s3 naming req's. Totally makes sense now. I'll update, rebase, and push corrections. Thank you!

Reply:
@nopcoder I was looking over s3 bucket naming rules and noticed s3 buckets must also adhere to these 2 specific requirements
- `Bucket names must not start with the prefix xn--.`
- `Bucket names must not end with the suffix -s3alias`

I noticed with the current regex iteration, users are able to make repo names that match these restrictions. Should I improve the regex to avoid these 2 specific prefix and suffix restrictions?

Reply:
FYI code has been updated so that wording now states that 3 characters is the minimum character limit for Repository ID.

Reply:
Thanks, @DataDavD, I would open a different issue for the extra validation as the enforcement should be first implemented on the backend. 
Compatibility is part of S3 gateway, if all S3 clients or SDKs do not validate 'xn--' and '-s3alias' - lakefs can support it, as the limitation is from the underlying storage.